### castling
uint8 side

### side ###
uint8 NoCastle  =  48  # '0' (ascii zero)
uint8 KingSide  =  75  # 'K'
uint8 QueenSide =  81  # 'Q'
